src/redisearch_rs/c_entrypoint/query_error_ffi/cbindgen.toml

```diff
@@ -72,3 +72,6 @@ include = ["c_ffi_utils", "query_error"]
 
 [export.rename]
 "OpaqueQueryError" = "QueryError"
+
+[export]
+include = ["QueryWarningCode"]
```

Comment:
Since you do a `pub use .. QueryWarningCode` in `src/redisearch_rs/c_entrypoint/query_error_ffi/src/lib.rs`, i don't think this line is needed.